### ISSUE TITLE
JSON file for raining

### ISSUE THREAD

**Issue by BBuniverse** [xg416/DATUM#3]
Hi, just wondering where can we find the train_info.json and test_info.json for make_lmdb.py and train_DATUM_dynamic.py?

Thanks in advance.

**Comment by xg416** [comment]
Hi, train_info.json and test_info.json are generated by make_lmdb.py, please downloaded our updated ATSyn-dynamic dataset and run make_lmdb.py to get them

**Comment by BBuniverse** [comment]
Thank u for ur reply.

I tried to download the updated dataset again, but still no info JSON inside. However, line 14 in make_lmdb.py still needs the test_info.json before making the lmdb dataset.

I noticed that there are some param JSON files inside the turb_param. However, the key name is 'level' which mismatches the 'turb_level' in line 58 inside the datase_LMDB_train.py. Also, these JSON files missing the key, 'h' 'w' 'length', in line 97 as well.
